bsd.go

```diff
@@ -7,23 +7,23 @@ package gopass
 #include <unistd.h>
 #include <stdio.h>
 
-int getch() {
-        int ch;
+char getch(int termDescriptor) {
+        char ch;
         struct termios t_old, t_new;
 
-        tcgetattr(STDIN_FILENO, &t_old);
+        tcgetattr(termDescriptor, &t_old);
         t_new = t_old;
         t_new.c_lflag &= ~(ICANON | ECHO);
-        tcsetattr(STDIN_FILENO, TCSANOW, &t_new);
+        tcsetattr(termDescriptor, TCSANOW, &t_new);
 
-        ch = getchar();
+        read(termDescriptor, &ch, sizeof(ch));
```

Comment:
There's also `fgetc` and a bunch of other variants, but I think those require me to wrap the descriptor in a `FILE*`, and I don't know if we want to do that.